fmriprep/run_workflow.py

```diff
@@ -128,6 +136,8 @@ def main():
         subject_list = [op.basename(subdir)[4:] for subdir in glob.glob(
             op.join(settings['bids_root'], 'sub-*'))]
 
+    logger.info("subject data is " + pprint.pformat(subject_list))
```

Comment:
Same as in the other PR, this is just a list so pprint is overkill the problem and adds one more import.